rel_bball_analytics/statistics/summary.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+
+from .data_source import get_player_stats
+
+EXCLUDED_COLUMNS = [
+    "player_id",
+    "team_id",
+    "game_id",
+]
+
+
+def get_summary_stats(player_ids: list, season: int):
+    """Return dataframe with summary stats for each player"""
+    summary_stats = []
+    for player_id in player_ids:
+        player_stats = get_player_stats(player_id=player_id, season=season)
+
+        if player_stats is None:
+            continue
+
+        player_summary_stats = get_player_summary_stats(stats=player_stats)
+        summary_stats.append(player_summary_stats)
+
+    return pd.DataFrame(summary_stats)
+
+
+def get_player_summary_stats(stats: pd.DataFrame):
+    """Return obj with summary stats for given id and season"""
+    summary_stats = stats.drop(EXCLUDED_COLUMNS, axis=1).mean(axis=0, numeric_only=True)
```

Comment:
Converting result to dict will also convert numpy and nan types to python types so that no additional cleanup/checks need to be made in `players.styles`.